index.html

```diff
@@ -0,0 +1,218 @@
+          Nigeria
+        </p>
+      </div>
+    </header>
+    <main>
+      <section id="about">
+        <div class="about">
+          <div class="about-left">
+            <img
+              src="./images/profile.JPG"
+              alt="A picture of Temitoyin Ayorinde"
+            />
+          </div>
+          <div class="about-right">
+            <hr />
+            <h1 class="about-heading">About me</h1>
+            <p>
+              I'm a sotfware developer with experience in creating applications
+              using HTML, CSS and JavaScript. I like learning about other
+              platforms such as the backend because it gives a wider view on how
+              to approach problems. I have a good understanding of version
+              control and team collaboration tools such as Pivotal Tracker. I
+              spend my spare time reading, playing basketball, swimming or
+              generally just having fun. I have a keen interest in languages,
+              currently I speak English, French, Hausa and Yoruba. In the coming
+              years I plan to increase the list with probably Spanish. If you
+              want to work with me, feel free to
+              <a href="#contact">get in touch</a>
+            </p>
+            <a class="anchor-btn" href="#portfolio">Go to portfolio</a>
+          </div>
+        </div>
+      </section>
+      <section>
+        <div class="proficiency">
+          <div class="proficiency-header">
+            <h2>Proficiency</h2>
+          </div>
+          <hr />
+          <div class="categories">
+            <div class="front-end">
+              <h3>front end</h3>
+              <ul>
+                <li>HTML5</li>
+                <li>CSS3</li>
+                <li>JavaScript</li>
+                <li>Bootstrap</li>
+              </ul>
+            </div>
+            <div class="back-end">
+              <h3>Back end</h3>
+              <ul>
+                <li>Node.js</li>
+                <li>Postman</li>
+                <li>Express</li>
+              </ul>
+            </div>
+            <div class="version-control">
+              <h3>Version control</h3>
+              <ul>
+                <li>Git</li>
+                <li>Github</li>
+              </ul>
+            </div>
+            <div class="data-base">
+              <h3>Database</h3>
+              <ul>
+                <li>MySQL</li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- <section id="portfolio">
+        <div class="portfolio">
+          <div class="portfolio-heading">
+            <h4>Portfolio</h4>
+          </div>
+          <div class="portfolio-list">
+            <div class="portfolio_ls"></div>
+            <div class="portfolio_ls"></div>
+            <div class="portfolio_ls"></div>
+            <div class="portfolio_ls"></div>
+          </div>
+        </div>
+      </section> -->
```

Comment:
Can you remove all commented code please.

Reply:
Done